.github/workflows/create_additional_release_tag.yaml

```diff
@@ -30,7 +25,13 @@ jobs:
         ARTIFACT_IDS=('google-cloud-shared-dependencies' 'api-common' 'gax')
         for ARTIFACT_ID in "${ARTIFACT_IDS[@]}"
         do
-          VERSION=$(grep "${ARTIFACT_ID}" versions.txt | cut -d':' -f2)
-          git tag ${ARTIFACT_ID}/$VERSION ${{ github.event.inputs.releaseTag }}
-          git push origin ${ARTIFACT_ID}/$VERSION
+          VERSION=$(grep "^${ARTIFACT_ID}:" versions.txt | cut -d':' -f2 | tr -d '[:space:]')
```

Comment:
Can you format the indentation to follow https://google.github.io/styleguide/shellguide.html#loops

Reply:
Thanks for the reference! Updated.